server/export.go

```diff
@@ -94,6 +94,9 @@ func ExportCmd(appExporter types.AppExporter, defaultNodeHome string) *cobra.Com
 				Validator: tmproto.ValidatorParams{
 					PubKeyTypes: exported.ConsensusParams.Validator.PubKeyTypes,
 				},
+				Version: tmproto.VersionParams{
+					AppVersion: exported.ConsensusParams.GetVersion().GetAppVersion(),
+				},
```

Comment:
[question] I'm curious why upstream Cosmos SDK didn't already have this diff. Without this diff, how do other Cosmos SDK chains stop + export + start a chain?

Reply:
I have no idea